qiskit/providers/honeywell/api/rest/job.py

```diff
@@ -48,5 +48,5 @@ def __init__(self, session, job_id):
 
     def status(self):
         """Return the status of a job."""
-        url = self.get_url('status') + '?websocket=true'
+        url = f'{self.get_url("status")}?websocket={"false" if self.session.proxies else "true"}'
```

Comment:
Yiou can fix the python 3.5 compat by just doing:
```suggestion
        url = "{url}?websocket={proxies}".format(url=self.get_url("status"), proxies="false" if self.session.proxies else "true")
```

Reply:
I am aware of how to avoid f-strings.  :)

Thank you for the fast review!

Reply:
All f-strings removed.